src/main/java/org/recordrobotics/charger/RobotMap.java

```diff
@@ -27,4 +27,8 @@ public class DriveBase {
 		public static final int RIGHT_MIDDLE_MOTOR_PORT = 0;
 		public static final int RIGHT_FRONT_MOTOR_PORT = 3;
 	}
+
+	public class Claw {
+		public static final int CLAW_MOTOR_PORT = 2;
```

Comment:
don't need the word claw here, as the variable will only be called with class `Claw`